samples/cpp/hello_query_device/main.cpp

```diff
@@ -71,14 +72,33 @@ void print_any_value(const ov::Any& value) {
         slog::info << std::get<1>(values);
         slog::info << " }";
         slog::info << slog::endl;
+    } else if (value.is<ov::hint::PerformanceMode>()) {
+        auto mode = ov::util::property_to_string(value.as<ov::hint::PerformanceMode>());
+        slog::info << (mode.empty() ? "\"\"" : mode) << slog::endl;
```

Comment:
since the condition in the 'if' and the actual type in the  as are the same, can we simply make this function a template?
with @apankratovantonp and @nkogteva